jerry-core/include/jerryscript-compiler.h

```diff
@@ -110,6 +120,10 @@ extern "C"
 #define JERRY_UNLIKELY(x) (x)
 #endif /* !JERRY_UNLIKELY */
 
+#ifndef JERRY_VLA
+#define JERRY_VLA(type, name, size) type name[size]
+#endif /* JERRY_VLA */
```

Comment:
style: comment needs a ! prefix, i.e. `/* !JERRY_VLA */`

Reply:
good point

Reply:
done and also fixes GCC